Tests/LibJS/test-invalid-unicode-js.cpp

```diff
@@ -0,0 +1,76 @@
+/*
```

Comment:
Not sure if this is how we want to test things like this (and about the name).
But this is the most explicit way I could think of.